infra/frontend/app-config/staging.tf

```diff
@@ -20,4 +20,8 @@ module "staging_config" {
   instance_desired_instance_count = 4
   instance_scaling_min_capacity   = 4
   instance_scaling_max_capacity   = 20
+
+  service_override_extra_environment_variables = {
+    NEW_RELIC_ENABLED = "false"
+  }
```

Comment:
The current configuration sets the default to true, and then only staging to false. I don't think that's intentional?

Reply:
oh 🤦🏼‍♀️ idk if you were even ready for a re-review yet

Reply:
Kay!